Resources/Prototypes/_NF/Entities/Objects/Consumable/Food/meals.yml

```diff
@@ -275,3 +275,134 @@
     tags:
     - Vegetable
     - Fruit
+  
+- type: entity
+  name: poached pear
+  parent: FoodBowlBase
+  id: FoodMealPoachedPears
+  description: The classiest use for a can of wine.
+  components:
+  - type: FlavorProfile
+    flavors:
+      - fruity
+      - alcohol
+      - pear
+  - type: Sprite
+    sprite: _NF/Objects/Consumable/Food/bowl.rsi
+    layers:
+    - state: bowl
+    - state: poachedpear
+  - type: SolutionContainerManager
+    solutions:
+      food:
+        maxVol: 25
+        reagents:
+        - ReagentId: Nutriment
+          Quantity: 12
+        - ReagentId: Vitamin
+          Quantity: 5
+        - ReagentId: Flavorol
+          Quantity: 8
+  - type: Tag
+    tags:
+    - Vegetable
+    - Fruit
+
+- type: entity
+  name: belle pear
```

Comment:
As far as I can tell, this dish is properly called "pears belle Helene", or "pears belle" for short, named after the operette _La belle Hélène_ "The beautiful Helen".

The ID should definitely be either `FoodMealBellePears` or, probably better, `FoodMealPearsBelle`.

Reply:
Went with "pears belle helene" and FoodMealPearsBelleHelene if only for consistency with caps - no accents, but I've seen at least a few English language sources that skip them as well.  Seems reasonable to me.